webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js

```diff
@@ -83,7 +83,7 @@ const ActionsBar = ({
       href={`/hosts/${hostId}`}
       key="prev-version"
     >
-      {__('Previous UI')}
+      {__('Old UI')}
```

Comment:
Maybe we can go even further? :)
```suggestion
      {__('Legacy UI')}
```

Reply:
At this point the "legacy" is still the default. So you can also say
```suggestion
      {__('Default UI')}
```

But this naming is hard.

Reply:
As discussed on maintainers call, lets make it legacy UI.